styles/login.css

```diff
@@ -0,0 +1,25 @@
+.navbars{
+    background-color: var(--secondary);
+}
+.form-container{
+    padding: 0 1rem;
+}
+.form{
+    max-width: 400px;
+    border: 2px solid var(--secondary);
+    margin: 5rem auto;
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    align-items: center;
+    gap: 1rem;
+    padding: 1rem 0.5rem;
+}
+.brand-name{
+    font-family: 'Amarante', cursive;
```

Comment:
you can add this in root styles and keep this as variable. It will help you ahead

Reply:
I am just using it for brand name and in footer name that's why I gave it directly but will change it later.